### ISSUE TITLE
con't mount ipfs

### ISSUE THREAD

**Issue by CJentzsch** [ipfs/go-ipfs#1702]
`ERRO[09:17:08:000] error mounting: fusermount: exit status 1 fusermount: exit status 1  module=core/commands`

even with `sudo ipfs mount -f <existingPath>`

**Comment by jbenet** [comment]
will have to give more data than this to debug. maybe stop by #ipfs on freenode.

**Comment by edouard-lopez** [comment]
### Command 

Fails after ~25s:

```
sudo ipfs mount -f /tmp/test-mount
Error: fuse failed to access mountpoint /ipns
```

### Daemon 

```
22:38:41.993 ERROR core/comma: error mounting: %!s(<nil>) fusermount: exit status 1 mount_unix.go:219
```

**Comment by whyrusleeping** [comment]
i think we need a little UX change here. `ipfs mount` tries to mount both ipfs and ipns, the -f and -n options just change the mountpoint for that particular fs

**Comment by jbenet** [comment]
I've been wanting a revamp of the `mount` command for a while. specifically would like these to work: 

```sh
# mount "/ipfs" into local "~/ipfs"
ipfs mount /ipfs ~/ipfs

# mount "/ipns" into local "~/ipns"
ipfs mount /ipns ~/ipns

# mount "/ipfs/<hash>/a/b/c" into local "./c"
ipfs mount /ipfs/<hash>/a/b/c ./c
```

**Comment by IntenseRave** [comment]
`ipfs mount`
it worked before
but now after rebooting the system and restarting the daemon command
it is printing this error

> 18:44:46.136 ERROR       node: error mounting: mountpoint does not exist: /ipfs mount_unix.go:89
18:44:46.136 ERROR       node: error mounting: mountpoint does not exist: /ipns mount_unix.go:93

i am new to ipfs plz help if you can

**Comment by Stebalien** [comment]
@IntenseRave Your issue is probably unrelated to this bug. You probably just need to create the `/ipfs` and `/ipns` directories. If that's not sufficient, please open a new issue (so we don't add too much noise here).

**Comment by kallisti5** [comment]
```
kallisti5@desktop ~ :( $ ipfs --api="/ip4/192.168.1.10/tcp/45001" mount
Error: fuse failed to access mountpoint /ipfs

kallisti5@desktop ~ :( $ ls -lah /ipfs /ipns
/ipfs:
total 0
drwxrwxr-x  1 kallisti5 kallisti5   0 Aug  8 16:40 .
dr-xr-xr-x. 1 root      root      230 Aug  8 16:40 ..

/ipns:
total 0
drwxrwxr-x  1 kallisti5 kallisti5   0 Aug  8 16:40 .
dr-xr-xr-x. 1 root      root      230 Aug  8 16:40 ..
```

**Comment by Stebalien** [comment]
It sounds like you're running IPFS on a different machine?

**Comment by kallisti5** [comment]
@Stebalien indeed.   If there is a limitation there, ipfs might need logic to alert the user that remote connectivity for fuse mounts doesn't work in that configuration.

**Comment by Stebalien** [comment]
Well, it "works", it'll just mount `/ipfs` on the remote machine...
